concent_api/core/templates/admin/base.html

```diff
@@ -0,0 +1,28 @@
+{% extends "admin/base.html" %}
+{% load static %}
+{% load admin_tags %}
+{% block extrastyle %}
+	<link rel="stylesheet" type="text/css" href="{% static '/admin/css/soft_shutdown.css' %}"/>
+{% endblock %}
+{% block messages %}
+
+	{% is_shutdown_mode_enabled as is_shutdown_mode_enabled %}
+	{% result_uploaded_subtasks_amount as result_uploaded_subtasks_amount %}
+	{% active_subtasks_amount as active_subtasks_amount %}
+	{% shutdown_mode_state as shutdown_mode_state %}
+
+	{% if is_shutdown_mode_enabled %}
+
+		<div class="shutdown_mode {{ shutdown_mode_state }}">
+			{% if shutdown_mode_state == 'active_subtasks' %}
+				Shutting down. Active Subtasks: {{ active_subtasks_amount }}. Passive Subtasks with RESULT_UPLOADED: {{ result_uploaded_subtasks_amount }}
```

Comment:
it is not very readable. I think it would be better to have each entry in separate line.
And maybe have the numbers **bolded**?

Reply:
Done.